.travis.yml

```diff
@@ -27,6 +27,7 @@ install:
 
 script:
   - docker-compose exec web bash -c "CI=TRUE GENERATE_REPORT=true rake test:all"
+  - cat ./spec/TEST-Teaspoon-Result.xml >> output2.xml
```

Comment:
Should maybe this be unnecessary since anything starting with TEST and ending with `.xml` will get compiled into output.xml?